### PR TITLE
[4.0] Remove non working custom help

### PR DESCRIPTION
**Pull request by brianteeman**
In J3 there was the ability to add your own link (labelled custom help) on the help dropdown menu.

In J4 we no longer have a help dropdown - we have a dashboard instead which is constructed in a completely different way. In addition in Joomla 4 we have the ability to place a much more prominent custom support link as a module in the sidebar.

As a result this PR removes the code related to creating the non-working custom help link.

To test just make sure the admin menu works as it did before and that the remaining options in the admin menu module (show new and show help) still work.

Someone can always add this functionality back into Joomla at a later date - but it definitely wouldnt use this code as explained above and imho better to remove the non-working code and module options

Pull Request for Issue ##26050

### CODE REVIEW

File: administrator/components/com_menus/presets/alternate.xml [hunk 1]
```diff
@@ -547,14 +547,6 @@
 			scope="help"
 		/>
 
-		<menuitem
-			type="url"
-			target="_blank"
-			title="MOD_MENU_HELP_SUPPORT_CUSTOM_FORUM"
```
Comment (Quy) [inline administrator/components/com_menus/presets/alternate.xml:553]: Delete language string?

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 4d0924c38b1933a45b18a750c482a2bf4101ffc0<br /><br />Tested successfully in Beta6-dev of 24 December using PHP 7.4.13.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31774">issues.joomla.org/tracker/joomla-cms/31774</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 4d0924c38b1933a45b18a750c482a2bf4101ffc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31774">issues.joomla.org/tracker/joomla-cms/31774</a>.</sub>

**Comment by infograf768** [comment]
The custom forum works fine when using the alternative main  menu.

Screen Shot 2020-12-24 at 09 14 51

Can't we try to have it also work in joomla main menu instead of deleting all?

**Comment by brianteeman** [comment]
> Someone can always add this functionality back into Joomla at a later date - but it definitely wouldnt use this code as explained above and imho better to remove the non-working code and module options

**Comment by chmst** [comment]
On the admin login page, the module login_support contains links to support sites shich can be customised. One of the hidden sectrets which needs documentation.

**Comment by chmst** [comment]
RTC

**Comment by brianteeman** [comment]
No point in keeping this as rtc and not merging it for over a month so closed

**Pull request closed without merging**